ftml/README.md

```diff
@@ -36,8 +36,16 @@ You can use this as a dependency by adding the following to your `Cargo.toml`:
 ftml = "0.4"
```

Comment:
Should probably update this in the readme, lol

Reply:
I haven't deployed any new versions of ftml to crates.io

Reply:
Oops, nevermind then.